tests/test_client.py

```diff
@@ -52,25 +64,25 @@ async def handle_filtered_messages() -> None:
             await client.publish(topic_unfiltered, 2)
 
 
-async def test_client_unsubscribe(os_and_version: str) -> None:
-    topic_header = os_and_version + "/tests/asyncio_mqtt/unsubscribe/"
+async def test_client_unsubscribe() -> None:
+    topic_header = TOPIC_HEADER + "unsubscribe/"
     topic1 = topic_header + "1"
     topic2 = topic_header + "2"
     event = anyio.Event()
```

Comment:
Maybe I'm blind but what's the purpose of this `event`? We set it but never await it.